raidprotect/build.rs

```diff
@@ -1,6 +1,7 @@
 fn main() -> Result<(), Box<dyn std::error::Error>> {
     rosetta_build::config()
         .source("fr", "./locales/fr.json")
+        .source("en", "./locales/en.json")
         .fallback("fr")
```

Comment:
The fallback language should be the same as the default one. 

```suggestion
        .source("en", "./locales/en.json")
        .source("fr", "./locales/fr.json")
        .fallback("en")
```